### PR TITLE
Update /upgrades/vision with premerge copy

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Update /upgrades/vision with pre-merge copy
- Calls out The Merge
- Basic updates to sharding content

## Related Issue
- #6281 
- #6590

### CODE REVIEW

File: src/intl/en/page-upgrades-vision.json [hunk 1]
```diff
@@ -21,34 +23,34 @@
...
-  "page-upgrades-vision-scalability-desc-3": "The shard chains upgrade will spread the load of the network into 64 new chains. This will give Ethereum room to breathe by reducing congestion and improving speeds beyond the current 15-45 transactions per second limit.",
-  "page-upgrades-vision-scalability-desc-4": "And even though there will be more chains, this will actually require less work from validators - the maintainers of the network. Validators will only need to 'run' their shard and not the entire Ethereum chain. This makes nodes more lightweight, allowing Ethereum to scale and remain decentralized.",
+  "page-upgrades-vision-scalability-desc-3": "Sharding upgrades will spread the data storage requirements across the entire network, no longer requiring every node to hold 100% of the data. Although this doesn't directly address scaling the execution of transactions, this problem is being addressed directly by layer 2 rollup solutions.",
```
Comment (corwintines) [inline src/intl/en/page-upgrades-vision.json:26]:
```suggestion
  "page-upgrades-vision-scalability-desc-3": "Sharding upgrades will spread the data storage requirements across the entire network, no longer requiring every node to hold 100% of the data. Although this doesn't directly address scaling the execution of transactions, this problem is being addressed directly by <a href=\"/layer-2/\">layer 2</a> rollup solutions.",
```

Just adding link out to layer 2 page

**Review by corwintines — COMMENTED** [review]
Looks good, just added a suggestion for linking out to the layer 2 page.

File: src/intl/en/page-upgrades-vision.json [hunk 2]
```diff
@@ -1,18 +1,20 @@
 {
   "page-upgrades-vision-2014": "View a 2014 blog post detailing proof of stake",
   "page-upgrades-vision-2021": "View a 2021 blog post on the Ethereum roadmap evolution",
+  "page-upgrades-vision-2022": "View a 2022 blog post: The Hitchhikers Guide to Ethereum",
```
Comment (minimalsm) [inline src/intl/en/page-upgrades-vision.json:4]: These key names are horrible 😅

File: src/intl/en/page-upgrades-vision.json [hunk 3]
```diff
@@ -21,34 +23,34 @@
...
+  "page-upgrades-vision-security-desc-8": "Ethereum's security model also needs to change because of the introduction of sharding. The Beacon Chain will coordinate all of the validators who will be responsible for asserting that all data has been made available, but every node will no longer be required to hold the entire history of the chain. A new role is also anticipated, known as a dedicated block builder, who will work alongside block proposers (validators) for efficient and safe block production. Proof-of-stake is a prerequisite to sharding.",
+  "page-upgrades-vision-security-desc-10": "Staking also means you don't need to invest in elite hardware to participate directly in consensus. This should encourage more people to become a validator, increasing the network's decentralization and decreasing the attack surface area.",
   "page-upgrades-vision-security-staking": "Stake ETH",
-  "page-upgrades-vision-security-validator": "You can become a validator by staking your ETH.",
+  "page-upgrades-vision-security-validator": "You can become a validator by staking your ETH, but remember, you don't need any ETH to run your own node.",
```
Comment (minimalsm) [inline src/intl/en/page-upgrades-vision.json:36]: I don't like this addition. The original acts more as a CTA for people to become stakers.

File: src/intl/en/page-upgrades-vision.json [hunk 4]
```diff
@@ -21,34 +23,34 @@
...
-  "page-upgrades-vision-security-desc-10": "Staking also means you don't need to invest in elite hardware to 'run' an Ethereum node. This should encourage more people to become a validator, increasing the network's decentralization and decreasing the attack surface area.",
+  "page-upgrades-vision-security-desc-8": "Ethereum's security model also needs to change because of the introduction of sharding. The Beacon Chain will coordinate all of the validators who will be responsible for asserting that all data has been made available, but every node will no longer be required to hold the entire history of the chain. A new role is also anticipated, known as a dedicated block builder, who will work alongside block proposers (validators) for efficient and safe block production. Proof-of-stake is a prerequisite to sharding.",
```
Comment (minimalsm) [inline src/intl/en/page-upgrades-vision.json:33]: I'd argue that this change has very little to do with security 🤔

File: src/intl/en/page-upgrades-vision.json [hunk 5]
```diff
@@ -21,34 +23,34 @@
...
-  "page-upgrades-vision-scalability-desc-4": "And even though there will be more chains, this will actually require less work from validators - the maintainers of the network. Validators will only need to 'run' their shard and not the entire Ethereum chain. This makes nodes more lightweight, allowing Ethereum to scale and remain decentralized.",
+  "page-upgrades-vision-scalability-desc-3": "Sharding upgrades will spread the data storage requirements across the entire network, no longer requiring every node to hold 100% of the data. Although this doesn't directly address scaling the execution of transactions, this problem is being addressed directly by layer 2 rollup solutions.",
+  "page-upgrades-vision-scalability-desc-4": "Rollups need cheap storage on layer 1 though to be most effective. Data sharding will give Ethereum room to breathe by maximizing the efficiency on rollups, enabling exponential improvements beyond the current 15-45 transactions per second limit. Validators will only need to sample the data to make sure it all has been made available.",
```
Comment (minimalsm) [inline src/intl/en/page-upgrades-vision.json:27]: Don't think the last sentence is relevant if we're just trying to explain that sharding makes rollups cheaper.

```suggestion
  "page-upgrades-vision-scalability-desc-4": "Rollups need cheap storage on layer 1 though to be most effective. Data sharding will give Ethereum room to breathe by maximizing the efficiency on rollups, enabling exponential improvements beyond the current 15-45 transactions per second limit.",
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Looking good :-) few suggestions below. 

I'd love to chat on the 'data sharding' phrasing we're using on this (and some other) Merge PRs. I don't think rebranding 'sharding' -> 'data sharding' makes any sense from a technical or UX perspective.

File: src/intl/en/page-upgrades-vision.json (hunk 3 above)
Comment (wackerow) [reply src/intl/en/page-upgrades-vision.json:36]:
```suggestion
  "page-upgrades-vision-security-validator": "You can become a validator by staking your ETH.",
```

File: src/intl/en/page-upgrades-vision.json (hunk 5 above)
Comment (wackerow) [reply src/intl/en/page-upgrades-vision.json:27]: DAS is how/why they won't have to store all of the data in order to make it cheaper
```suggestion
  "page-upgrades-vision-scalability-desc-4": "Rollups need cheap storage on layer 1 though to be most effective. Data sharding will give Ethereum room to breathe by maximizing the efficiency on rollups, enabling exponential improvements beyond the current 15-45 transactions per second limit.",
```

File: src/intl/en/page-upgrades-vision.json [hunk 6]
```diff
@@ -14,31 +15,31 @@
...
-  "page-upgrades-vision-scalability-desc-4": "And even though there will be more chains, this will actually require less work from validators - the maintainers of the network. Validators will only need to 'run' their shard and not the entire Ethereum chain. This makes nodes more lightweight, allowing Ethereum to scale and remain decentralized.",
+  "page-upgrades-vision-scalability-desc-3": "Sharding upgrades will spread the data storage requirements across the entire network, no longer requiring every node to hold 100% of the data. Although this doesn't directly address scaling the execution of transactions, this problem is being addressed directly by <a href=\"/layer-2/\">layer 2</a> rollup solutions.",
+  "page-upgrades-vision-scalability-desc-4": "Rollups need cheap storage on layer 1 though to be most effective. Data sharding will give Ethereum room to breathe by maximizing the efficiency on rollups, enabling exponential improvements beyond the current 15-45 transactions per second limit.",
```
Comment (minimalsm) [inline src/intl/en/page-upgrades-vision.json:19]:
```suggestion
  "page-upgrades-vision-scalability-desc-4": "Rollups need cheap storage on layer 1 though to be most effective. Sharding will give Ethereum room to breathe by maximizing the efficiency on rollups, enabling exponential improvements beyond the current 15-45 transactions per second limit.",
```

**Review by minimalsm — APPROVED** [review]
Removed one instance of 'data sharding' but let's get this in and deployed :-)

**Pull request merged**